contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -289,6 +293,10 @@ GetTDSRequest(bool *resetProtocol)
 			return NULL;
 		}
 
+		/* Enable statement timeout. Note we add this function here to
+		 * include the time taken by the protocol in the timeout  */
```

Comment:
In Postgres we follow the following patter for a multi-line comment:
```
/*
 * Enable statement timeout. Note we add this function here to
 * include the time taken by the protocol in the timeout.
 * /
```
Also it should end with full-stop